README.md

```diff
@@ -63,8 +63,10 @@ staramr search -o out --pointfinder-organism salmonella *.fasta
   * [Dependencies](#dependencies)
 - [Output](#output)
   * [summary.tsv](#summarytsv)
+  * [detailed_summary.tsv](#detailedsummarytsv)
```

Comment:
I believe the link should be to `(#detailed_summarytsv)`

Reply:
Whoops, I thought that would have worked too, apparently not, fixed that in 0ab9cf2